config.json

```diff
@@ -1,6 +1,6 @@
 [
```

Comment:
Bitte einmal das Dokument auf Kohärenz bei der Groß- und Kleinschreibung von "Social Media" überprüfen. In Zeile 25 wird es bspw. groß geschrieben, in Zeile 46 wird "media" klein geschrieben.

Gleiches gilt für "text" und "image"